### ISSUE TITLE
reget for interrupted downloads

### ISSUE THREAD

**Issue by doyle52** [MiniGlome/Archive.org-Downloader#25]
If for some reason a download session is interrupted or if fetched images are incomplete, any attempt to to continue downloading only starts from zero again. 
It would be great if a reget feature for incomplete downloads like the one's we are used to with wget/curl could be added. 
Would be just great for people with an unstable and/or slow internet connection. 
Thanks!

**Comment by darnn** [comment]
Yes, this is something that happens to me all the time, unfortunately, so I would love that as well.

**Comment by MiniGlome** [comment]
This is tricky because it means the script first has to scan for files with the same name as the book you want to download. However multiple books could have the same name and so the script would have no way to determine wether the files are from a previously downloaded book with the same name or the partially downloaded book. 🤔

**Comment by farOverNinethousand** [comment]
> This is tricky because it means the script first has to scan for files with the same name as the book you want to download.

Then this would need to be addressed first --> Put unique identifier into foldername see also #26.
